### ISSUE TITLE
Final predictions are not printing both targets

### ISSUE THREAD

**Issue by mhurley** [udacity/aind2-nlp-capstone#1]
## Problem:

The final predictions print one target correctly but the other is all zeros and results in <PAD>

```
print(' '.join([y_id_to_word[np.argmax(x)] for x in y[0]]))
```

> `<PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD> <PAD>`

## Fix: 

```
print(' '.join([y_id_to_word[x[0]] for x in y[0]]))
```
    
> new jersey est parfois calme pendant l' automne et il est neigeux en avril

## Suggestion

I would also suggest to label the target and predicted to easier understand the output. Updated code below.

```
print('Sample 1:')
print('Predict: ' + ' '.join([y_id_to_word[np.argmax(x)] for x in predictions[0]]))
print('Target : ' + 'Il a vu un vieux camion jaune')
print('\nSample 2:')
print('Predict: ' + ' '.join([y_id_to_word[np.argmax(x)] for x in predictions[1]]))
print('Target : ' + ' '.join([y_id_to_word[x[0]] for x in y[0]]))
```

**Comment by mhurley** [comment]
It also seems to be predicting and printing twice for one of the strings.

Duplicate print of second prediction

`print(logits_to_text(model.predict(x[:1])[0], y_tk))`

`print(' '.join([y_id_to_word[np.argmax(x)] for x in predictions[1]]))`

**Comment by Brok-Bucholtz** [comment]
I can't find the line `print(logits_to_text(model.predict(x[:1])[0], y_tk))`.  Do you have the newest version of the code?

**Comment by mhurley** [comment]
Guess not, or maybe I had added it? Don't see it anymore either way, my bad.

Thanks for taking a look at this so quickly.

**Comment by Brok-Bucholtz** [comment]
Welcome!

**Comment by mhurley** [comment]
Hey Brock,

I see this was closed but there is no code update that fixed the problem. Can you make sure you pushed that?

Matt
